### ISSUE TITLE
Full hill color range for a specific area

### ISSUE THREAD

**Issue by jidanni** [der-stefan/OpenTopoMap#144]
I have a new idea for Opentopomap tiles.

Opentopomap looks great when the full range of hill colors are within
the screen. E.g., World view, or Indian plains to Mount Everest.

Alas, in the usual case, say the area we are looking at contains only
elevation 600 to 1200 m., or even less, then we are only looking at just
a small subset of all those colors and thus missing out...

Yes, one could say "well 1200 m. is not exactly ice covered mountains"
but anyway, you get the idea.

The problem is this whole concept of providing the full range of colors
for whatever is in the viewport runs contrary to the usual tile server
concept.

In fact no matter what weird WMTS address or short HTTP Expires time you
use, it will still be a waste. Even http://600-1200.bla... would be
dumb.

Therefore it seems the only way to present it to the user would be
one-shot views:
```
http://full-color-range.open-bla-bla-bla.org/#map=17/24.1817/120.8660
```

**Comment by der-stefan** [comment]
Hi Dan,

there is a dynamic map that seems to deliver what you are asking for: http://en-us.topographic-map.com/maps/24.1586802,120.7480704,24.3257923,120.9059429/

If we find out a similar way (drawing live on a canvas in the web browser), this would be a nice little feature also for OpenTopoMap.

**Comment by jidanni** [comment]
Thanks!
Alas, another anonymous site without any contact info.

**Comment by der-stefan** [comment]
You are right. I tried to contact the domain holder years ago...

**Comment by jidanni** [comment]
Actually I did manage to reach him at
```
$ whois topographic-map.com | grep ...
Registrant Email: webmaster@tessanet.fr
Admin Email: guillaume.vk@free.fr
```

**Comment by Trachsou** [comment]
Hello,

I was wonderig if you got any further in your research? I'm looking for a map like this for month now. Because i find nothing that is as good as the map from Topographic-map.com for relief coloring.

Thanks

**Comment by der-stefan** [comment]
No, unfortunately I can't spend any time on OpenTopoMap in the next months. Of course, if somebody wants to join the project and push things forwards, I would be happy to merge your commits. ;-)

**Comment by jidanni** [comment]
No, @Trachsou,  [he](https://github.com/der-stefan/OpenTopoMap/issues/144#issuecomment-370731698) only responded to me once.

**Comment by der-stefan** [comment]
This feature will be possible with our new vector maps! The clients have full access to the raw elevation profile and render the color range on their own. Dynamic scaling or even changing the color map will be "just" an issue for the webinterface.
